packages/circuit-ui/components/ColorInput/ColorInput.spec.tsx

```diff
@@ -13,10 +13,11 @@
  * limitations under the License.
  */
 
-import { describe, expect, it } from 'vitest';
+import { describe, expect, it, vi } from 'vitest';
 import { createRef } from 'react';
+import { userEvent } from '@storybook/test';
```

Comment:
`userEvent` should be imported from `../../util/test-utils.js`.

Reply:
Updated.